movie-browser/src/App.js

```diff
@@ -1,9 +1,48 @@
 import React from 'react';
+import Card from './Card.js'
+
+// dummy data
+const movie = {
```

Comment:
Store the dummy data on another file. It shouldn't be in the `App` file.

Reply:
Good lesson! 👌